dev/README_RELEASE_AIRFLOW.md

```diff
@@ -583,15 +583,14 @@ Check if sources are the same as in the tag:
 ```bash
 cd X.Y.Zrc1
 tar -xvzf *-source.tar.gz
-pushd apache-airflow-X.Y.Zrc1
+pushd apache-airflow-X.Y.Z
 diff -r airflow "${SOURCE_DIR}"
-popd && rm -rf apache-airflow-X.Y.Zrc1
+popd && rm -rf apache-airflow-X.Y.Z
 ```
 
 The output should only miss some files - but they should not show any differences in the files:
 
 ```
-⌂6.50 [jarek:~/asf-dist/dev/airflow/2.6.2rc2/a] * 1 ‡ diff -r airflow ~/code/airflow/
```

Comment:
I think we need to keep just the command, no?

Reply:
That was what I did initially actually, but after thinking about it, I don't think we need it. It already mentions the "output", meaning it's the result of the commands above. This is the same pattern we have for the [shasum check](https://github.com/apache/airflow/blob/main/dev/README_RELEASE_AIRFLOW.md#sha512-sum-check) too.

Reply:
Is this acceptable, or do folks still want the command in here? @potiuk @hussein-awala?

Reply:
It's okay for me, LGTM

Reply:
I just realized that my first review was a comment and not an approve

Reply:
Haha no worries, thanks!